packages/combo-box/src/vaadin-combo-box-data-provider-mixin.js

```diff
@@ -81,20 +70,38 @@ export const ComboBoxDataProviderMixin = (superClass) =>
       ];
     }
 
+    constructor() {
+      super();
+
+      /**
+       * @type {DataProviderController}
+       * @private
+       */
+      this.__dataProviderController = new DataProviderController(this, {
+        size: this.size,
```

Comment:
Seems like this line is untested (all the tests still pass when I remove it).

Reply:
Thanks, I removed this line.